test/functional/apps/dashboard/group1/index.ts

```diff
@@ -28,7 +28,6 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
 
     // This has to be first since the other tests create some embeddables as side affects and our counting assumes
     // a fresh index.
-    loadTestFile(require.resolve('./empty_dashboard'));
```

Comment:
I removed the `empty_dashboard` test from `group1` because they were duplicated in `group6` so they were being run twice